nemo/collections/asr/data/vocabs.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import abc
+import itertools
+import string
+from typing import List
+
+import g2p_en
```

Comment:
Can you describe your choice of phoneme processing library?

Reply:
Sure: there are two well-made libs of G2P conversion: g2p_en and g2p. First one just promises better quality, it predicts conversion with seq2seq inference, vocabulary lookup and text normalizing.